pages/components/button.html

```diff
@@ -78,4 +78,150 @@ <h3 class="my-1p5">Components</h3>
               </ul>
             </nav>
           </aside>
-        <!-- aside left ends -->
\ No newline at end of file
+        <!-- aside left ends -->
+        <!-- main doc starts -->
+        <main class="main-docs">
+            <h1 class="mb-3">Button</h1>
+            <p class="main-text">Button is a tapping target that provides the user a simple way to trigger events like searching, submitting,etc .</p>
+        
+            <!--  button filled  starts  -->
+            <section class="section-component my2" id="button-filled">
+                <h2 class="my-2">Filled Primary Buttons</h2>
+                <div class="component-container">
+                <div class="flex justify-c-ctr alg-i-ctr gap1">
+                    
+                    <button class="btn">Default</button>
+                    <button class="btn error-btn-solid">Error</class=>
+                    <button class="btn primary-btn-solid">Primary</button>
+                    <button class="btn secondary-btn-solid">Secondary</button>
+                    <button class="btn sucess-btn-solid">Sucess</button>
+                    
+                </div>
+    
+                <pre class="pre-code">
+                  <code class="language-html">      
+                    &lt;button class="btn">Default&lt;/button>
+                    &lt;button class="btn btn-error-solid">Error&lt;/button>
+                    &lt;button class="btn btn-primary-solid">Primary&lt;/button>
+                    &lt;button class="btn btn-secondary-solid">Secondary&lt;/button>
+                    &lt;button class="btn btn-success-solid">Success&lt;/button>
+                    
+                    
+                    
+                  </code>
+                </pre>
+            </div>
+            </section>
+              <!-- button filled  starts  -->
+
+            <!-- Outlined Primary Buttons Starts -->
+            <section class="section-component my2" id="button-outline">
+                <h2 class="my-2">Outlined Primary Buttons</h2>
+                <div class="component-container">
+                <div class="flex justify-c-ctr alg-i-ctr gap1">
+                    
+                    <button class="btn error-btn-outline">Error</button>
+                    <button class="btn primary-btn-outline">Primary</button>
+                    <button class="btn secondary-btn-outline">Secondary</button>
+                    <button class="btn sucess-btn-outline">Sucess</button>
+                    
+                </div>
+    
+                <pre class="pre-code">
+                  <code class="language-html">      
+                    &lt;button class="btn btn-error-outline">Error&lt;/button>
+                    &lt;button class="btn btn-primary-outline">Primary&lt;/button>
+                    &lt;button class="btn btn-secondary-outline">Secondary&lt;/button>
+                    &lt;button class="btn btn-success-outline">Success&lt;/button>      
```

Comment:
Similar issue here. I am not too sure if this is an issue.  btn-[type]-outline on lines 132 to 135, and [type]-btn-outline on lines 123 to 126. Pardon me if this was intended. I might be wrong here.